examples/item.json

```diff
@@ -57,6 +57,16 @@
         "satellite"
       ]
     },
+    {
+      "href": "https://maps.example.com/wms",
+      "rel": "wms",
+      "type": "text/xml",
+      "title": "RGB composite visualized through a WMS",
+      "wms:layers": [
+        "rgb"
+      ],
+      "wms:styles": []
```

Comment:
if the styles parameter is optional it would be probably best to completely leave it out if its empty?

Reply:
Doesn't really make a difference, I think, but I removed it.